udf/mrn_udf_highlight_html.cpp

```diff
@@ -295,6 +301,21 @@ MRN_API mrn_bool mroonga_highlight_html_init(UDF_INIT *init,
     info->query_mode.default_column = NULL;
   }
 
+  info->specify_tag.used = false;
+  if (args->arg_count == 4) {
```

Comment:
Does it work if multiple keywords specified like below?

```
SELECT mroonga_highlight_html('Mroonga is the Groonga based storage engine.',
                              'groonga', 'mroonga',
                              '<mark>' AS open_tag, '</mark>' AS close_tag) AS highlighted;
```

Reply:
Ah, I found the test for the multiple keywords.
But why they work as expected...?

Reply:
Thank you for your comment.

You are right. Currently, this function does not work if multiple keywords are specified.
Tests of this function are not executed now. Because this modification failed to build yet.

Reply:
I see. Thenk you for your reply!